stream-chat-android-compose/src/main/res/values/colors.xml

```diff
@@ -30,6 +30,8 @@
     <color name="stream_compose_error_accent">#FF3742</color>
     <color name="stream_compose_info_accent">#20E070</color>
     <color name="stream_compose_highlight">#FBF4DD</color>
+    <color name="stream_compose_show_more_overlay">#99000000</color>
```

Comment:
Slightly darker than the original light mode overlay color in order to improve contrast.